### DISCUSSION TITLE
Create text transcriptions from audio files

### DISCUSSION THREAD

**Discussion by ted-bunny** [Pustur/whatsapp-chat-parser-website#48] [Ideas]
Excellent software! 

It would be great if it were possible to generate a text transcription from each audio file, to be displayed right below the audio control of each recording identified during the parsing of the 'chat.txt' file.

I see great potential for this tool to support forensic and professional analyses, such as those conducted by law enforcement agents, including lawyers.

**Comment by Pustur** [comment]
Cool idea, but i think it would be way to complicated for me to implement it in this project.

Maybe an alternative would be too look for browser extensions / sites that do this instead.
Here's what chatGPT suggests:

1. **Transkriptor**: This extension transcribes audio and video files, including from embedded <audio> tags, and supports over 100 languages. It also allows exporting transcripts in various formats like TXT and SRT. Transkriptor offers features like speaker identification and translation, with a free trial for 90 minutes of transcription​.
2. **SpeechText.AI**: This extension enables transcription by uploading audio or video files, providing editable text and support for multiple languages. It includes features like speaker identification and natural language search within transcripts​.
3. **Otter.ai Chrome Extension**: Otter.ai can transcribe live meetings or audio content from websites. It includes speaker labeling and keyword searching in transcripts. This tool integrates seamlessly with platforms like Google Meet and Zoom, but it can also work with other audio content​.
4. **Fireflies.ai**: Primarily for meetings, Fireflies.ai can also transcribe audio and video content. It includes options for sharing, summarizing, and exporting transcripts. It supports GDPR compliance for secure data handling​.
